compiler/rustc_resolve/src/lib.rs

```diff
@@ -1227,16 +1228,19 @@ impl<'tcx> Resolver<'_, 'tcx> {
         // FIXME: remove `def_span` body, pass in the right spans here and call `tcx.at().create_def()`
         let def_id = self.tcx.create_def(parent, name, def_kind);
 
-        // Create the definition.
```

Comment:
This isn't actually deleted; rather, it's marked as removed in git and is consequently moved below (This is a result of moving `﻿self.tcx.untracked().source_span.push(span);` a few lines ahead)